test/wpt.ts

```diff
@@ -39,7 +39,7 @@ export function test(backend: Backend) {
   const limit = Infinity;
   let count = 0;
 
-  for (const testDir of [wptNodeTests]) {
+  for (const testDir of [wptTraversalTests, wptNodeTests]) {
```

Comment:
This is fine, but WPT tests don't really work atm. We'll need to write some separate basic tests for TreeWalker in `test/units`